framework/src/controller/action.ts

```diff
@@ -15,12 +15,13 @@
 import { strict as assert } from 'assert';
 import { actionWithModuleNameReg, moduleNameReg } from '../constants';
 import { RequestObject, VERSION } from './jsonrpc';
+import { ResponseObject, JsonRpcResult, ID, JsonRpcError } from './jsonrpc/types';
```

Comment:
you could import from above definition.
`import { ID, JsonRpcError, RequestObject, ResponseObject, JsonRpcResult, VERSION } from './jsonrpc';`